scripts/development/m3_stack/start_m3.sh

```diff
@@ -199,6 +199,9 @@ curl -vvvsSf -X POST localhost:7201/api/v1/namespace -d '{
           "aggregated": false
         }
       ]
+    },
+    "stagingState": {
+      "status": "INITIALIZING"
```

Comment:
This is a little odd, is it required to send the staging state now?

Reply:
Yeah, I went back and forth on adding some sugar to the namespace/add endpoint versus being explicit. Since these endpoints are mostly called by the operator, I decided to be explicit, but it's trivial to just set this in the endpoint itself. Thoughts?

Reply:
Yeah suppose that's ok, the thing is we have https://github.com/m3db/m3/blob/master/site/content/docs/how_to/cluster_hard_way.md which may need updating and may get rather annoying to use; don't know if this is a case we care about a lot or not though

Reply:
@ryanhall07 If you're not using the operator, you'll need to set the value yourself or leave it unset. I forgot to mention when responding to @arnikola that it's not actually required. It's only required when using etcd to retrieve namespaces in the Coordinator (which is something _existing_ clients will have to opt into via a specific migration at which point we can update all existing namespaces to "READY").

@arnikola Yeah, I have an AI to update the docs once all this lands. We can leave the docs as-is for the moment because not setting the staging state will not break current behavior (i.e. loading namespaces via coordinator yaml). I just updated this script so as to have the latest behavior in our dev scripts.

Reply:
K, mind commiting it in a provisional changelog entry just so we remember to do it?